package.json

```diff
@@ -57,6 +57,7 @@
     "eslint": "^5.6.1",
     "eslint-plugin-react": "^7.11.1",
     "expect.js": "0.3.1",
+    "husky": "^1.1.3",
```

Comment:
I'm still not sure I trust this but ok...as long as it doesn't modify anything outside of this project.